src/text-field.jsx

```diff
@@ -414,6 +414,10 @@ const TextField = React.createClass({
     }
   },
 
+  _getFontSize() {
+    return this.props.style && this.props.style.fontSize ? this.props.style.fontSize : 16;
```

Comment:
@oliviertassinari warning added without `babel-plugin-dev-expression`, as you have seen here https://github.com/callemall/material-ui/pull/2107